### PR TITLE
Bug 1314957 - Use centralized metastore

### PR DESCRIPTION
**Pull request by whd**
I tested that the longitudinal dataset was available from a new spark cluster and that `sqlContext.sql("show tables").show()` shows a bunch of tables. We could potentially remove parquet2hive from the standard install as well but I don't think it hurts to leave it.

### CODE REVIEW

File: ansible/envs/dev.yml [hunk 1]
```diff
@@ -4,3 +4,4 @@ telemetry_analysis_spark_emr_bucket: telemetry-spark-emr-2
 telemetry_analysis_public_bucket: telemetry-public-analysis-2
 telemetry_analysis_private_bucket: telemetry-private-analysis-2
 telemetry_analysis_anaconda_path: /mnt/anaconda2
+telemetry_analysis_metastore_stack_name: telemetry-metastore
```
Comment (whd) [inline ansible/envs/dev.yml]: Notably this differs from https://github.com/mozilla/emr-bootstrap-presto/blob/master/envs/default.yml#L22 but appears to better reflect reality.

Comment (robotblake) [inline ansible/envs/dev.yml]: Yeah. :\

**Comment by vitillo** [comment]
This is great :rage1:!

**Comment by vitillo** [comment]
Please redeploy.

**Comment by whd** [comment]
This was redeployed after PR #72.

**Pull request merged**